one-ui/ui/src/components/entities/entity-tiles.tsx

```diff
@@ -47,7 +41,25 @@ const EntityTiles: React.FC = () => {
           } catch (error) {
               let message = error;
               console.log('Error while fetching mapping artifacts', message);
-              //handleError(error);
+          }
+    }
+
+    const getMappingArtifactByMapName = async (entityName,mapName) => {
+        try {
+            let response = await axios.get(`/api/artifacts/mapping/${mapName}`);
+            
+            if (response.status === 200) {
+                let entArt = response.data;
```

Comment:
This is a "mapping", right? "entArt" implies it's an entity model artifact, but the URL indicates it's a mapping.

Reply:
Done.